### PR TITLE
hi, please have a look at all this goodness

### PR DESCRIPTION
**Pull request by abouyer**
of course, feel free to edit and comment !

Best regards
Antoine, suggested by Francky

### CODE REVIEW

File: bin/chrootBuddy.sh [hunk 1]
```diff
@@ -123,6 +124,7 @@ fi
 
 log "Configuring fstab for shm"
 sed -i  -e 's/#\/dev\/shm/\/dev\/shm/' "/etc/schroot/$NAME/fstab"
+echo "/work       /work        none    rw,bind         0       0" >> "/etc/schroot/$NAME/fstab"
```
Comment (xroumegue) [inline bin/chrootBuddy.sh]: This sounds as a user specific choice..

File: bin/pclBuddy.sh [hunk 2]
```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+# helper to setup the pointclouds libraries
+# http://www.pointclouds.org/
+
+SOURCE="$(pwd)/srcs"
+POC="$(pwd)/lidar-poc"
+
+# Git remote definition
+PCL_REMOTE="https://github.com/PointCloudLibrary/pcl.git"
+PCL_DEMATHIF_REMOTE="https://github.com/demathif/pcl.git"
```
Comment (xroumegue) [inline bin/pclBuddy.sh:10]: Oh noooo !

File: bin/pclBuddy.sh [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+function pcl_setup()
+{
+	echo "Checking sources folder...."
+	if [ ! -d "$SOURCE"/pcl ]; then
+		echo "$SOURCE/pcl does not exist... cloning a fresh repo";
+		[ -d "$SOURCE" ] || mkdir -p "$SOURCE"
+		cd $SOURCE && git clone "$PCL_REMOTE"
+		cd "$SOURCE"/pcl
+		git remote add github $PCL_DEMATHIF_REMOTE
```
Comment (xroumegue) [inline bin/pclBuddy.sh:22]: oh noooooooooooo!

File: bin/chrootBuddy.sh [hunk 4]
```diff
@@ -24,6 +25,7 @@ while true; do
 		-r | --release) RELEASE=$2; shift 2;;
 		-a | --arch) ARCH=$2; shift 2;;
 		-P | --packages) PACKAGES_LIST+=" $2"; shift 2;;
+		-e | --extra) EXTRA_LIST=$2; shift 2;;
```
Comment (xroumegue) [inline bin/chrootBuddy.sh]: This extra option seems compulsory .. but it's not initialized.. sadly.

Comment (abouyer) [reply bin/chrootBuddy.sh]: no this extra parameter is optional. PKG_EXTRA is filled if --extra is provided. But indeed, there is a warning if extra is not set. This is going to be fixed.

File: bin/chrootBuddy.sh (hunk 1 above)
Comment (abouyer) [reply bin/chrootBuddy.sh]: outdated. this line is removed in another patch

**Pull request merged**